golang/go-cloud-run-hello-world/main.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Google LLC
```

Comment:
I updated this sample from the original to better align with Cloud Run sample practices, including more error handling and use of the metadata client library.